App.tsx

```diff
@@ -1,118 +1,11 @@
-/**
```

Comment:
No changes should be there on `App.tsx` file at all. please revert the change to this file

Reply:
Done